server/punishments.js

```diff
@@ -604,6 +604,7 @@ const Punishments = new (class {
 			Punishments.ips.set(ip, punishment);
 		}
 		const [punishType, id, expireTime, reason, ...rest] = punishment;
+		// @ts-ignore
```

Comment:
ideally, the ignores should all say `cast to ID` or something, so when those files subsequently get refactored to `.ts`, we know why.